cmd/config/cmddocs/commands/docs.go

```diff
@@ -21,6 +21,16 @@ var CatExamples = `
     # unwrap Resource config from a directory in an ResourceList
     ... | kustomize config cat`
 
+var CompletionShort = `Install shell completion for kustomize commands and flags`
+var CompletionLong = `
+Install shell completion for kustomize commands and flags.
```

Comment:
can this say what the consequences of the _install_ are?
local artifacts?
when would i want/need to reinstall?

Reply:
Done